babel.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = (api) => {
-	api.cache(true);
-
+	//api.cache(true);
```

Comment:
Since this works for some folks projects can you wrap the `api.cache(true);` call in an if statement? Something like:

```js
if (api && api.cache) {
  api.cache(true);
}
```

Reply:
done, sorry for the delay